### PR TITLE
Lai rgalrong

### PR DESCRIPTION
**Pull request by LinguList**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

Concept list must be added for a lexibank dataset, where we currently draft the revision.

### CODE REVIEW

**Comment by LinguList** [comment]
@MuffinLinwist  and @CarolinHu would you be available to review this PR quickly? We would like to resubmit at some point soon, so it would be helpful to have this already covered. We do not expect many problems, since the concept list is mainly based on Sagart's list from 2019.

**Comment by MuffinLinwist** [comment]
> @MuffinLinwist and @CarolinHu would you be available to review this PR quickly? We would like to resubmit at some point soon, so it would be helpful to have this already covered. We do not expect many problems, since the concept list is mainly based on Sagart's list from 2019.

Yes! I'll add it as part of the tasks for the next two days..

**Comment by LinguList** [comment]
@laiyunfan, you can see the current data in EDICOTR with a link that makes it easy to browse the data when using this link:

```
https://digling.org/edictor/?file=lairgyalrong.tsv&preview=500&basics=DOCULECT|CONCEPT|TOKENS|COGIDS&publish=true
```

I think we may argue that this is enough for the time being?

**Comment by LinguList** [comment]
I'd then make a version for the current state of the data, and that would be it?

**Comment by laiyunfan** [comment]
Yes, I think that would be it. Then I will do a final check of the manuscript and the response, and will finally submit it.

File: concepticondata/conceptlists/Lai-2023-291.tsv [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+Lai-2023-291-4	149	the otter	15	OTTER	317	the otter
+Lai-2023-291-5	208	that	78	THAT	947	that
+Lai-2023-291-6	2	all	98	ALL	962	all
+Lai-2023-291-7	265	here	136	HERE		here
+Lai-2023-291-8	372	to burn [transitive]	141	BURN (SOMETHING)		to burn [transitive]
+Lai-2023-291-9	92	the hoof	152	HOOF	264	the hoof
+Lai-2023-291-10	158	red	156	RED	1007	red
+Lai-2023-291-11	14	black	163	BLACK	1005	black
+Lai-2023-291-12	16	to blow (of wind)	175	BLOW (OF WIND)	1738	to blow (of wind)
+Lai-2023-291-13	17	the body hair (hair or fur)	189	HAIR (BODY)	266	the body hair (hair or fur) // the hair
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Lai-2023-291.tsv:14]: Perhaps, [HAIR OR FUR](https://concepticon.clld.org/parameters/2678) is better for this one.

File: concepticondata/conceptlists/Lai-2023-291.tsv [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+Lai-2023-291-47	135	the mud	640	MUD	46	the mud
+Lai-2023-291-48	295	the ashes	646	ASH		the ashes
+Lai-2023-291-49	156	the rain	658	RAIN (PRECIPITATION)	10	the rain
+Lai-2023-291-50	162	the river	666	RIVER	30	the river
+Lai-2023-291-51	163	the road	667	ROAD	38	the road
+Lai-2023-291-52	164	the root	670	ROOT	375	the root
+Lai-2023-291-53	170	the sand	671	SAND	44	the sand
+Lai-2023-291-54	43	early	672	EARLY	1018	early
+Lai-2023-291-55	134	the mouth	674	MOUTH	84	the mouth
+Lai-2023-291-56	250	the shit	676	SHIT (DEFECATE)	151	the shit
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Lai-2023-291.tsv:57]: The current mapping refers to the action. I believe [FAECES (EXCREMENT)](https://concepticon.clld.org/parameters/675) would be a more suitable mapping for this gloss.

File: concepticondata/conceptlists/Lai-2023-291.tsv [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+Lai-2023-291-75	7	the barley (tibetan or highland)	932	BARLEY	411	the barley (tibetan or highland)
+Lai-2023-291-76	12	the bird	937	BIRD	326	the bird
+Lai-2023-291-77	15	the blood	946	BLOOD	129	the blood
+Lai-2023-291-78	229	the water	948	WATER	47	the water
+Lai-2023-291-79	238	the wind	960	WIND	9	the wind
+Lai-2023-291-80	242	the woman	962	WOMAN	174	the woman
+Lai-2023-291-81	161	right	1019	RIGHT	711	right
+Lai-2023-291-82	425	to yawn	1028	YAWN		to yawn
+Lai-2023-291-83	76	good	1035	GOOD	1052	good
+Lai-2023-291-84	80	the hair (of the head)	1040	HAIR	75	the hair (of the head)
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Lai-2023-291.tsv:85]: [HAIR (HEAD)](https://concepticon.clld.org/parameters/2648) would be a more suitable mapping.

File: concepticondata/conceptlists/Lai-2023-291.tsv [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+Lai-2023-291-85	117	light (of weight)	1052	LIGHT (WEIGHT)	1015	light (of weight)
+Lai-2023-291-86	233	the wheat	1077	WHEAT	410	the wheat
+Lai-2023-291-87	359	the work	1086	WORK (ACTIVITY)		the work
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Lai-2023-291.tsv:88]: The current mapping referes to the action. [WORK (LABOUR)](https://concepticon.clld.org/parameters/984) is a more suitable mapping.

File: concepticondata/conceptlists/Lai-2023-291.tsv [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+Lai-2023-291-129	210	thick	1244	THICK	980	thick
+Lai-2023-291-130	188	small	1246	SMALL	965	small
+Lai-2023-291-131	42	the ear	1247	EAR	81	the ear
+Lai-2023-291-132	49	the eye	1248	EYE	79	the eye
+Lai-2023-291-133	9	the belly	1251	BELLY	96	the belly
+Lai-2023-291-134	97	the house	1252	HOUSE	494	the house
+Lai-2023-291-135	84	the head	1256	HEAD	74	the head
+Lai-2023-291-136	239	the wing	1257	WING	298	the wing
+Lai-2023-291-137	183	to sing	1261	SING	1184	to sing
+Lai-2023-291-138	90	high / tall	1265	HIGH	968	high / tall
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Lai-2023-291.tsv:139]: [HIGH OR TALL](https://concepticon.clld.org/parameters/2790) perhaps is a more suitable mapping.

File: concepticondata/conceptlists/Lai-2023-291.tsv [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+Lai-2023-291-139	138	narrow	1267	NARROW	977	narrow
+Lai-2023-291-140	168	the salt	1274	SALT	61	the salt
+Lai-2023-291-141	81	the hand	1277	HAND	107	the hand
+Lai-2023-291-142	227	to vomit	1278	VOMIT	1535	to vomit
+Lai-2023-291-143	379	to fall	1280	FALL		to fall
+Lai-2023-291-144	312	the fat (grease)	1282	FAT (FROM ANIMALS)		the fat (grease)
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Lai-2023-291.tsv:145]: `the fat (grease)` has been previously mapped as [FAT (ORGANIC SUBSTANCE)](https://concepticon.clld.org/parameters/323) in our concept lists.

File: concepticondata/conceptlists/Lai-2023-291.tsv [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+Lai-2023-291-176	204	straight	1404	STRAIGHT	1003	straight
+Lai-2023-291-177	137	the name	1405	NAME	687	the name
+Lai-2023-291-178	50	far	1406	FAR	974	far
+Lai-2023-291-179	370	to breathe	1407	BREATHE		to breathe
+Lai-2023-291-180	85	to hear	1408	HEAR	1682	to hear
+Lai-2023-291-181	173	to see	1409	SEE	1471	to see
+Lai-2023-291-182	153	to play	1413	PLAY	1703	to play
+Lai-2023-291-183	414	to squeeze	1414	SQUEEZE		to squeeze
+Lai-2023-291-184	213	to think (reflect)	1415	THINK (REFLECT)	1726	to think (reflect)
+Lai-2023-291-185	409	to sit	1416	SIT		to sit
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Lai-2023-291.tsv:186]: Entry for `HUANG_1992_1820` column is empty. Just to confirm, is this not missing?

File: concepticondata/conceptlists/Lai-2023-291.tsv [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+Lai-2023-291-281	147	old (of person)	2112	OLD (AGED)	1058	old (of person)
+Lai-2023-291-282	136	the nail (fingernail or claw)	2128	CLAW OR NAIL	113	the nail (fingernail or claw)
+Lai-2023-291-283	133	the mouse or rat	2139	MUROID (MOUSE OR RAT)	320	the mouse or rat
+Lai-2023-291-284	125	to marry (a man marries a woman)	2164	MARRY (AS MAN)	1578	to marry (a man marries a woman)
+Lai-2023-291-285	70	the front (front side)	2194	FRONT (PART)	712	the front (front side)
+Lai-2023-291-286	365	thin (object)	2249	THIN (OF SHAPE OF OBJECT)		thin (object)
+Lai-2023-291-287	45	the earthworm	2350	EARTHWORM	363	the earthworm
+Lai-2023-291-288	38	the dream	2374	DREAM	699	the dream
+Lai-2023-291-289	93	horizontal	2376	HORIZONTAL	1001	horizontal
+Lai-2023-291-290	430	wide	2531	BROAD		wide
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Lai-2023-291.tsv:291]: Perhaps, [WIDE](https://concepticon.clld.org/parameters/1243) or [BROAD OR WIDE](https://concepticon.clld.org/parameters/2652) is a more suitable mapping?

**Review by MuffinLinwist — APPROVED** [review]
@LinguList and @laiyunfan, my review of this PR is finished. Just minor comments on some mappings.

File: concepticondata/conceptlists/Lai-2023-291.tsv (hunk 1 above)
Comment (LinguList) [reply concepticondata/conceptlists/Lai-2023-291.tsv:14]: The Chinese concept is maximally unspecified, but in this case, the base list links already to body hair, so we better just follow that mapping.

File: concepticondata/conceptlists/Lai-2023-291.tsv (hunk 2 above)
Comment (LinguList) [reply concepticondata/conceptlists/Lai-2023-291.tsv:57]: Yes, this is a wrong mapping.

File: concepticondata/conceptlists/Lai-2023-291.tsv (hunk 7 above)
Comment (LinguList) [reply concepticondata/conceptlists/Lai-2023-291.tsv:186]: These are new entries. This is correct.

**Comment by LinguList** [comment]
Thanks, @MuffinLinwist, good catches, I'll look into those later.

**Comment by LinguList** [comment]
My updates should have addressed these cases now.

**Comment by LinguList** [comment]
I'll thus merge now. I also fixed an error in Huang, which is useful, thanks @MuffinLinwist !

**Comment by laiyunfan** [comment]
Thank you very much!

**Pull request merged**